azure-iot-device/azure/iot/device/provisioning/internal/action.py

```diff
@@ -0,0 +1,52 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
```

Comment:
Should the contents of this file be relocated and renamed to match the new pipeline architecture, something similar to how they are in the iothub library?

Reply:
yes I think this file will itself change or not be there when its pipeline ..so i am not modifying it now

---
In reply to: [282961176](https://github.com/Azure/azure-iot-sdk-python-preview/pull/81#discussion_r282961176) [](ancestors = 282961176)